.circleci/config.yml

```diff
@@ -356,7 +356,7 @@ commands:
                   --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
                   --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
                   --device model=blueline,version=28,locale=en,orientation=portrait \
-                  --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
```

Comment:
Why was the --no-record-video introduced at all, do you have the context? I haven't added it in Flutter config and it's really convenient to see what's going on with the test actually

Reply:
to save cost I assume 😄